### PR TITLE
[ECHOJS-118] Delete method subscribe_to_market 

### PR DESCRIPTION
**Pull request by alesPoz**

### CODE REVIEW

File: src/echo/subscriber.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ class Subscriber extends EventEmitter {
 		this._api = api;
 
 		await this._wsApi.database.setSubscribeCallback(this._onRespond.bind(this), true);
-
+		//
```
Comment (nicklatkovich) [inline src/echo/subscriber.js]: remove

File: src/echo/subscriber.js [hunk 2]
```diff
@@ -445,6 +445,28 @@ class Subscriber extends EventEmitter {
 		return null;
 	}
 
+	/**
```
Comment (nicklatkovich) [inline src/echo/subscriber.js]: remove

File: src/echo/subscriber.js [hunk 3]
```diff
@@ -458,6 +480,13 @@ class Subscriber extends EventEmitter {
 		const updates = messages.filter((msg) => {
 			// check is object id
 			if (isObjectId(msg)) {
+				// _updateOrder -> return order type - push to orders = { type, order }
```
Comment (nicklatkovich) [inline src/echo/subscriber.js]: remove

**Review by nicklatkovich — CHANGES_REQUESTED** [review]
1) Try to resolve conflicts more accurate. When you was resolving conflicts you return functions, that was removed in another one branch
2) Remove this method in `Subscriber` too
3) Remove `Subscriber#subscribers.market`

File: src/echo/subscriber.js (hunk 2 above)
Comment (alesPoz) [reply src/echo/subscriber.js]: I don't understand, should i delete this  /**  thing? If I do that then a disaster will happened with this code)

**Pull request merged**